ogre2/src/Ogre2SelectionBuffer.cc

```diff
@@ -192,9 +192,12 @@ void Ogre2SelectionBuffer::CreateRTTBuffer()
   const_cast<Ogre::CompositorPassSceneDef *>(scenePass)->mVisibilityMask =
       IGN_VISIBILITY_SELECTABLE;
 
-  // buffer to store render texture data
-  size_t bufferSize = Ogre::PixelUtil::getMemorySize(width, height, 1, format);
+  // buffer to store render texture data. Ensure it's at least 4 bytes
+  size_t bufferSize = std::min<size_t>(
```

Comment:
should this be `std::max`?

Reply:
Cr*p you're right.

I made that mistake, fixed it; then changed branches to submit the PR and reapplied the fix. I must've applied the wrong version.

I just pushed another version